StringCatalogEnum/Tests/StringCatalogEnumTests/File.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  File.swift
+//  
+//
+//  Created by Anish Gupta on 11/24/23.
+//
+
+/*
+import Foundation
+import XCTest
```

Comment:
As I mentioned in Slack, you need to understand Quick and Nimble. When you work on a new project, you can't just say "I'm not familiar with the frameworks you are using, so let's just use XCTest".

Besides, we do have an example of Quick and Nimble: https://github.com/superarts/XStringCatalog/blob/master/StringCatalogEnum/Tests/StringCatalogEnumTests/StringCatalogEnumTests.swift